supabase/migrations/20230809162431_schema.sql

```diff
@@ -1,3 +1,16 @@
+create table "public"."dynamic_links" (
```

Comment:
Something up with the migration. Each time we create a migration it should be in a new file that is timestamped. So the user management migration shouldn't be touched in future PRs. The work to add the new stuff should be in new migrations.

Reply:
Leave migrations, name them accordingly.